src/components/ui/CustomSelect.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import isEqual from 'lodash/isEqual';
+
+export default class CustomSelect extends React.Component {
```

Comment:
A couple of things here:
- Select always the first option if you type in the search input
- No results template
- Toggle button, Is this possible?